compute/cloud-client/src/main/java/compute/snapshotschedule/CreateSnapshotSchedule.java

```diff
@@ -17,6 +17,8 @@
 package compute.snapshotschedule;
 
 // [START compute_snapshot_schedule_create]
+import static com.google.cloud.compute.v1.ResourcePolicySnapshotSchedulePolicyRetentionPolicy.OnSourceDiskDelete.KEEP_AUTO_SNAPSHOTS;
```

Comment:
Enums should be prefixed with their class name. Modify the import statement to:

`import com.google.cloud.compute.v1.ResourcePolicySnapshotSchedulePolicyRetentionPolicy. OnSourceDiskDelete;`

Reply:
Thank you! Fixed